languages/java/exercises/concept/strings/.meta/design.md

```diff
@@ -0,0 +1,22 @@
+## Learning objectives
+
+- Know of the existence of the `String` type.
+- Know how to create a string.
+- Know of some basic string methods (like finding the index of a character at a position, or returning a part the string).
+- Know how to do basic string formatting.
+
+## Out of scope
+
+- Using standard or custom format strings.
+- Memory and performance characteristics.
+- Strings can be enumerated.
+
+## Concepts
+
+- `strings`: know of the existence of the `String` type; know of some basic functions (like looking up a character at a position, or slicing the string); know how to do basic string formatting.
```

Comment:
``- `strings`:`` ... Confused here the back tick enclosed text normally refers to something that is actually code. I don't believe Java has any class called `strings`. I cannot offer correction as I am not quite sure of the intention.

Generally though, I would advise being careful do differentiate between ``String`` and a "string" or "strings". To someone that already knows about Java string handling they will be able to infer the intention , but someone that is only learning could well be confused by the switching between "String", "string", "strings", `` `string` ``, `` `strings` ``, `` `String` `` and `` `Strings` ``.

This applies throughout the PR.

Reply:
I used the same approach I saw in other languages. As I can see the concepts (like basics, strings and so on) are put inside back ticks even if they are not something strictly relative to code